sass/public.scss

```diff
@@ -0,0 +1,235 @@
+@import '/sass/variables.scss';
+
+/**********************************/
+
+
+/*********** Nav Bar CSS **********/
+
+
+/**********************************/
+
+.navbar {
+    position: relative;
+    transition: .5s;
+    z-index: 999;
+    a {
+        text-decoration: unset;
+    }
+    &__navbar-collapse {
```

Comment:
I think that can be broken down to one more level `
&__navbar {  
&-collapse {
...
}
}`